packages/pn-commons/src/hooks/__test__/useBlocker.test.tsx

```diff
@@ -2,7 +2,7 @@ import type { Transition } from 'history';
 import React from 'react';
 import { UNSAFE_NavigationContext } from 'react-router-dom';
 
-import { renderHook } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react';
```

Comment:
Now you can replace the @testing-library/react with the import from the local test-utils.
Do it for all imports